homeassistant/components/flux_led/light.py

```diff
@@ -333,6 +331,12 @@ def turn_on(self, **kwargs):
         if brightness is None:
             brightness = self.brightness
 
+        if hs_color is None or (hs_color[0] == 0 and hs_color[1] == 0):
+            hs_color = (1, 1, 1)
+        
+        if brightness is None or brightness == 0:
+            brightness = 100
+            
```

Comment:
Couple of issues here.

1.  Brightness is only passed to flux_led when a change in brightness occurs in the front end.  Other wise it's none.  It's checked by flux_led on line 331 and set to the old or current value of brightness if it is none.  By line 337 it will never be none.
2.  brightness can't be set to 0 by the front end as far as I'm aware.  The lowest it goes is 1.
3. brightness is a scale of 1-255 not 0-100